sdk/src/quotes.ts

```diff
@@ -262,7 +296,7 @@ export const getCrosschainQuote = async (
   if (quote.error) {
     return quote as QuoteError;
   }
-  return quote as CrosschainQuote;
+  return { ...quote, minRefuelAmount } as CrosschainQuote;
```

Comment:
do we need `minRefuelAmount` passed here? i was under the impression we would fetch this value with `getMinRefuelAmount` from the app. Additionally this could delay the quote being fetched for a bit

Reply:
Yeah good point ive pushed a change to move it out into its own function exclusively